packages/sources/src/addSourceWizard/hardcodedComponents/gcp/costManagement.js

```diff
@@ -125,6 +127,20 @@ export const Dataset = () => {
 
 export const AssignAccess = () => {
     const intl = useIntl();
+    const [ googleAccount, setGoogleAccount ] = useState();
+
+    useEffect(() => {
+        const errorMessage = intl.formatMessage({
+            id: 'cost.gcp.noAccount',
+            defaultMessage: 'There is an error with loading of the account address. Please go back and return to this step.'
+        });
+
+        getSourcesApi().getGoogleAccount().then((data) => setGoogleAccount(data?.data?.[0]?.payload || errorMessage))
+        .catch((e) => {
+            console.error(e);
```

Comment:
Is this error log deliberate?

Reply:
Yeah. It should happen only on some network errors.